### ISSUE TITLE
Markdown preview fonts are not readable

### ISSUE THREAD

**Issue by cdswyda** [microsoft/vscode#195156]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.83.0 (user setup)
- OS Version:  Windows 11  22H2 22621.2283

Steps to Reproduce:

1.  Use any dark themes
2.  Open any md file, use Command "Markdown: Open Locked Preview to the Side"
3.  The next result will be obtained as shown below, where you can see that the text color is indistinguishable
  ![image](https://github.com/microsoft/vscode/assets/13060680/5d253c58-6a8c-4467-9afb-2e252184dbef)

When debugging, you can see that `--vscode-editor-foreground is not defined` .

Here's a gif of the procedure, I've disabled all plugins.
![10 09](https://github.com/microsoft/vscode/assets/13060680/08b86dce-2106-4918-818d-c65c21773c60)

After I switched themes, the text can be read. But when I open it again, it still has the same problem.

![10 09-2](https://github.com/microsoft/vscode/assets/13060680/a7f5c66f-a9fa-4695-a6d1-59a870cec152)

Detailed version information from VS code 
```
Version: 1.83.0 (user setup)
Commit: e7e037083ff4455cf320e344325dacb480062c3c
Date: 2023-10-03T16:12:16.321Z
Electron: 25.8.4
ElectronBuildId: 24154031
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Windows_NT x64 10.0.22621
```

**Comment by mjbvz** [comment]
> When debugging, you can see that --vscode-editor-foreground is not defined 

This would explain the issue but I can't reproduce the issue

Does this reproduce in the [latest VS Code insiders build](https://code.visualstudio.com/insiders/) with all [extensions disabled](https://github.com/microsoft/vscode/wiki/Performance-Issues#run-with-extensions-disabled)?

**Comment by cdswyda** [comment]
The insider version can't reproduce this.
This is a strange issue, I've disabled the extension but it still comes up, so I'm not sure how I should locate the problem.

**Comment by mjbvz** [comment]
Thanks for checking! My best guess is that your normal VS Code was corrupted somehow. Sometimes antivirus programs can do this. Either that or the insiders version has fixed the issue

Since it doesn't reproduce in insiders though I'm going to close this issue as resolved

**Comment by cdswyda** [comment]
I have attempted to reinstall, but it did not resolve the problem. I even manually deleted all the local files and directories associated with VSCode after uninstalling, but the issue persisted.

After performing a clean installation, everything was functioning properly. However, the problem arose immediately after I logged in and synchronized my settings. Therefore, I suspect that the issue may lie within my configuration files. Upon further investigation, I discovered that the markdown.preview.fontFamily setting was the cause of the problem. Restoring it to the default value resolved the issue with the markdown preview. I hope this information can assist others who are encountering similar difficulties.

This was a personal issue I encountered, but nevertheless, I appreciate your assistance.
